scripts/regressor_finder.py

```diff
@@ -340,7 +344,7 @@ def mercurial_to_git(revs):
             for bug_introducing_commit in prev_bug_introducing_commits
         )
         logger.info(
-            f"Already classified {len(prev_bug_introducing_commits)} commits..."
+            "Already classified %s commits...", len(prev_bug_introducing_commits)
```

Comment:
The value here is an integer:
```suggestion
            "Already classified %d commits...", len(prev_bug_introducing_commits)
```